docs/en/guides/sahi-tiled-inference.md

```diff
@@ -280,7 +280,7 @@ Example for batch prediction:
 from sahi.predict import predict
 
 predict(
-    model_type="yolov8",
+    model_type="yolo11",
```

Comment:
yolo11 works but ultralytics would be more generic and future proof, wdyt @RizwanMunawar ?

```suggestion
    model_type="ultralytics",
```

Reply:
Yes, let me update it :)